pkg/restore/restore.go

```diff
@@ -1115,13 +1115,15 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 			// want to dynamically re-provision it.
 			return warnings, errs
 
-		case hasDeleteReclaimPolicy(obj.Object):
-			ctx.log.Infof("Dynamically re-provisioning persistent volume because it doesn't have a snapshot and its reclaim policy is Delete.")
```

Comment:
alternative would have been to call `fallthrough` at this line, has the same result but smaller diff.

However commenting out is also fine :+1:

Reply:
Also thought about that, but commenting it out is more obvious that this is disabled :)